### ISSUE TITLE
Service Fabric skeleton service - undefined

### ISSUE THREAD

**Issue by sytone** [sytone/personal-blog-comments#7]
https://blog.sytone.com/2021/01/04/service-fabric-skeleton-service/ 

 The occasional posts of a guy who plays with technology

**Comment by shuyih** [comment]
hi @sytone , do you have the sample code for this post? Thanks 
https://blog.sytone.com/2021/01/04/service-fabric-skeleton-service/

**Comment by sytone** [comment]
No, I have not pushed this to GitHub. You should be able to cut and paste from the blog entry.

**Comment by shuyih** [comment]
Thanks.

Regards,
Shuyi
